config/locales/in_person_proofing/en.yml

```diff
@@ -130,7 +130,13 @@ en:
         state_id_jurisdiction_hint: This is the state that issued your ID
         state_id_jurisdiction_prompt: '- Select -'
         state_id_number: ID number
-        state_id_number_hint: May include letters and numbers
+        state_id_number_hint_html: "May include letters, numbers, and the following
```

Comment:
Looking at this again, it looks, not great. But since the classes used are not for styling per se, and thus not super likely to change anytime soon, I think this is ok.